src/test/java/org/folio/logintest/CredentialExistenceTest.java

```diff
@@ -61,26 +56,25 @@ public static void setup(final TestContext context) {
       context.fail(e);
     }
```

Comment:
This try-catch is not needed, VertxUnitRunner automatically fails the test if an Exception is thrown.

Reply:
Yes.